operator/config/rbac/api/role.yaml

```diff
@@ -27,4 +27,10 @@ rules:
     verbs:
       - get
       - list
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - get
```

Comment:
@liranr23 Just adding a notification about this RBAC change, so the CSV needs to be regenerated/updated.